src/liquid/components/ld-accordion/ld-accordion.tsx

```diff
@@ -3,6 +3,7 @@ import { getClassNames } from '../../utils/getClassNames'
 import { getScrollParent } from '../../utils/scroll'
 
 /**
+ * @virtualProp {(event: LdAccordionSectionCustomEvent<boolean>) => void} onLdaccordionchange - handles event emmitted by accordion section
```

Comment:
Hmm, this is starting to get confusing. This is a prop that is React-specific and not available for the web component. I don't think that we should add it here.

Reply:
I have removed it.